buildSrc/src/main/kotlin/core/android.gradle.kts

```diff
@@ -0,0 +1,5 @@
+package core
+
+plugins {
+    id("com.android.application") apply false
```

Comment:
You need to include the plugin so gradle can statically detect (and not complain about) the android block. This is a performance gain, you do not want to apply this because where we are including all these scripts the plugin is already applied.